include/deal.II/grid/manifold_lib.h

```diff
@@ -283,29 +283,50 @@ class CylindricalManifold : public Manifold<dim,spacedim>
                        const double tolerance = 1e-10);
 
   /**
+   * Compute the cartesian coordinates for a point given in cylindrical
```

Comment:
-> Cartesian (upper case 'C' -- it's a proper name, named after the Latin name of Descartes).